src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -497,14 +501,32 @@ var DynamicTable = new Class({
     },
 
     selectRow: function(rowId) {
-        this.deselectAll();
         this.selectedRows.push(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    deselectRow: function(rowId) {
+        this.selectedRows.erase(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    selectRows: function(rowId1, rowId2) {
```

Comment:
the rest looks fine, one last question, should we regulate what happens when `rowId1 == rowId2`?

Reply:
I missed that edge case in my testing. I've added a fix for it.